File: w02/d02/gadi_gottlieb/ASSIGNMENT_FILES/homework/lib/person.rb
class Person

  attr_reader(:name, :age, :hometown, :strength, :hp)

  def initialize(name, age, hometown, strength)
    @name = name
    @age = age
    @hometown = hometown
    @strength = strength
    @hp = 100
  end

  def alive?()
    @hp > 0
  end

end

File: w10/d04/sean_jennings/classes/person.js
var person = function(name,age) {
  this.name = name,
  this.age = age,
  this.energy = 5,

  this.introduce = function() {
    if (this.energy > 0)
      return this.name;
  } else {
    return "";
  },

  this.walk = function() {
    if (this.energy > 0) {
      return true;
      this.energy--
    } if else (this.energy == 0) {
      return false;
    }
    this.energy--
  },

  this.sleep = function() {
    this.energy = 5;
  }
};
